### ISSUE TITLE
Hoàng Sa, Trường Sa belongs to Vietnamese

### ISSUE THREAD

**Issue by Nguyendiem99** [liodali/osm_flutter#478]
need to re-learn history

**Comment by canhth1** [comment]
yes, I come from Viet Nam. **Hoang Sa and Truong Sa belong to Vietnam**. History has recorded this fact.

**Comment by Nguyenbach28** [comment]
Hoang Sa and Truong Sa belong to Vietnam

**Comment by sonnh2410** [comment]
yes, I come from Viet Nam. Hoang Sa and Truong Sa belong to Vietnam. History has recorded this fact.
